src/platform/nrfconnect/ConnectivityManagerImpl.h

```diff
@@ -81,6 +85,44 @@ class ConnectivityManagerImpl final : public ConnectivityManager,
     // the implementation methods provided by this class.
     friend class ConnectivityManager;
 
+public:
+    // Generic network status checkers
+    bool IsIPv6NetworkEnabled()
+    {
+#if CHIP_DEVICE_CONFIG_ENABLE_THREAD
+        return IsThreadEnabled();
+#elif CHIP_DEVICE_CONFIG_ENABLE_WIFI
+        return IsWiFiStationEnabled();
+#endif
+    };
+
+    bool IsIPv6NetworkApplicationControlled()
```

Comment:
Is this used anywhere?

Reply:
Not really, I added it for completeness sake. But right, it's not needed for now.

Reply:
The same for `IsIPv6NetworkAttached()`